src/commands/lint/__test__/lint.test.ts

```diff
@@ -48,8 +46,6 @@ describe('lintImageryPaths', () => {
     }, Error('Trailing 0 in resolution of dataset name: auckland_2020_0.20m'));
   });
   it('Should Pass', () => {
```

Comment:
While its not exactly part of this PR, test names should really be unique.

Reply:
Fixed in the [main PR](https://github.com/linz/argo-tasks/pull/780).